### PR TITLE
chore (quickstarts/kit): Port quarkus to gradle (#1431)

### LINKED ISSUE #1431: [Gradle Quickstart] Port `quickstarts/maven/quarkus` to equivalent Gradle Quarkus quickstart
### Description

Port [`quickstarts/maven/quarkus`](https://github.com/eclipse/jkube/tree/master/quickstarts/maven/quarkus) quickstart to Gradle

- [ ] Generate a sample Gradle application from [https://code.quarkus.io/](https://code.quarkus.io/) and make changes as done in maven quickstart
- [ ] Add Kubernetes and OpenShift Gradle Plugins
- [ ] Verify application is deployed correctly on [minikube](https://minikube.sigs.k8s.io/docs/) using Kubernetes Gradle Plugin
- [ ] Verify application is deployed correctly on [Red Hat Developer Sandbox](https://developers.redhat.com/developer-sandbox/get-started) using OpenShift Gradle Plugin

### PR DESCRIPTION
**Pull request by anurag-rajawat**
## Description

Fixes #1431 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by anurag-rajawat** [comment]
@rohanKanojia shall we bump the quarkus version to 2.9.0.Final?

**Comment by rohanKanojia** [comment]
thanks, yes I think we should do it.

File: quickstarts/gradle/quarkus/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+BUILD SUCCESSFUL in 15s
+1 actionable task: 1 executed
+
+$ kubectl get pods                 
+NAME                       READY   STATUS    RESTARTS   AGE
+quarkus-689c7d4bcc-t2jfw   1/1     Running   0          56s
+
+$ minikube service quarkus --url
+http://192.168.49.2:30185
```
Comment (rohanKanojia) [inline quickstarts/gradle/quarkus/README.md:78]: Could you please add a small section about native mode as well?

**Review by rohanKanojia — APPROVED** [review]

File: quickstarts/gradle/quarkus/README.md (hunk 1 above)
Comment (anurag-rajawat) [reply quickstarts/gradle/quarkus/README.md:78]: How to do that?

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**